libfxrecord/src/net/message.rs

```diff
@@ -436,51 +140,64 @@ impl DownloadStatus {
 
 pub type ForeignResult<T> = Result<T, ErrorMessage<String>>;
 
-impl_message! {
+message_ty! {
```

Comment:
What does `ty` mean in `message_ty!`? Thank you? 😄

Reply:
`ty` is `type`

Reply:
is that a common convention? could it be `message_type`?

Reply:
Done.